### DISCUSSION TITLE
Decentralised StableCoin doubt

### DISCUSSION THREAD

**Discussion by Abhishek-2416** [Cyfrin/foundry-full-course-cu#1675] [General]
So as patrick was explaning about the threshlold value
where in person A deposits $100 ETH and when suppose it goes down to $74 it gives other an option to liqudate using their DSC
so my doubt is if this person liquidates someone elses position , how will he have enough DSC to liquadate back his own ETH

**Comment by EngrPips** [comment]
Hello @Abhishek-2416, here https://github.com/Cyfrin/foundry-full-course-f23/discussions/1625  is a discussion where we really talked about this subject. The bottom line is let's assume that is a flaw of the codebase which needs to be improved. You can maybe think of a better way to go about it and open an issue on this repo with the possible solution that you think would work accurately.
